src/style.css

```diff
@@ -0,0 +1,58 @@
+body {
+  margin: 0;
+}
+
+.main-header {
+  height: 60px;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.main-header__menu {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+li:first-of-type a {
+  margin-left: 0;
+}
+
+.main-header__menu-link {
+  display: block;
+  font-size: 12px;
+  font-family: Roboto;
+  font-style: normal;
```

Comment:
you really need font-style here?

Reply:
Are u shure, that margins doesnt needed? Check this again please